docker/src/main/java/brooklyn/entity/container/docker/DockerInfrastructureImpl.java

```diff
@@ -182,7 +182,7 @@ public String apply(@Nullable Entity input) {
                 .displayName("Docker Applications"));
 
         if (config().get(SDN_ENABLE) && config().get(SDN_PROVIDER_SPEC) != null) {
-            Entity sdn = addChild(EntitySpec.create(config().get(SDN_PROVIDER_SPEC))
+            Entity sdn = addChild(EntitySpec.create((EntitySpec<?>)config().get(SDN_PROVIDER_SPEC))
```

Comment:
We don't use the results of this cast for anything, and in fact the return type of `addChild(EntitySpec)` is just `Entity` so it should be removed